_maps/RandomZLevels/Reebe.dmm

```diff
@@ -0,0 +1,265 @@
+"a" = (/turf/open/indestructible/reebe_void,/area/reebe)
```

Comment:
Hmm, where would you say it should go? I'm not actually really sure. "generic" maybe?